superset/datasets/commands/importers/v0.py

```diff
@@ -284,16 +284,14 @@ class ImportDatasetsCommand(BaseCommand):
     """
 
     # pylint: disable=unused-argument
-    def __init__(
-        self, contents: Dict[str, str], *args: Any, **kwargs: Any,
-    ):
+    def __init__(self, contents: Dict[str, str], **kwargs: Any):
         self.contents = contents
         self._configs: Dict[str, Any] = {}
 
         self.sync = []
-        if kwargs.get("sync_columns"):
+        if kwargs['sync_columns']:
```

Comment:
Lines 292-294?

Reply:
Yep! That get should work now, I believe.

Reply:
I mean, it doesn't really change the functionality, but I think that's considered the preferred way to access a key's value in a dictionary (the changes I made).

Reply:
Indeed, it does not change the functionality, but my argument is that we should be making only necessary or clearly beneficial changes because we need to contribute this upstream. 

Your call. I'll hit the approve button.

Reply:
I'll make the change. Thanks Rebecca.